bounce/server/resource/membership.py

```diff
@@ -66,10 +88,24 @@ class DeleteMembershipRequest(metaclass=ResourceMeta):
     __params__ = {
         'type': 'object',
         'additionalProperties': False,
-        'required': ['user_id'],
+        'required': ['user_id', 'member_role', 'editor_id', 'editor_role'],
         'properties': {
             'user_id': {
-                'type': 'string',
+                'type': 'integer',
+            },
+        }
+    }
+
+    __body__ = {
```

Comment:
Currently we're getting members_role from the body when deleting, but should that be from the args or token since we are deleting?

Reply:
That's true.  We'll need to delete the user knowing his role.  I think what we should do in order to find his role is use the id of the user we want to delete, query the memberships table using his id, and get his role from there instead of getting it from the body.  It should look very similar to how we do it for checking the editor's role.  Can you think of how you can do that?